cmd/internal/daemon/cmd.go

```diff
@@ -44,6 +48,14 @@ var Cmd = &cobra.Command{
 }
 
 func init() {
+	Cmd.Flags().StringVar(
+		&nameFlag,
+		"name",
+		"",
+		"The name of the daemon. Used to identify this daemon in Akita Cloud.",
+	)
+	cobra.MarkFlagRequired(Cmd.Flags(), "name")
+
```

Comment:
I wonder if it makes sense using an automatically-generated name like we do for everything else.  I'd certainly prefer that users gave the daemon a meaningful name, but it might be a roadbump to force them to do so.